Assets/MixedRealityToolkit/InputSystem/Sources/InteractionInputSources.cs

```diff
@@ -603,7 +603,7 @@ private static bool TryGetReading<TReading>(SourceCapability<TReading> capabilit
         {
             if (capability.IsAvailable)
             {
-                Debug.Assert(capability.IsSupported);
+                 Debug.Assert(capability.IsSupported);
```

Comment:
Are there examples where it is valid for IsAvailable to be true and IsSupported to be false?  Spatial Mapping?

Reply:
I don't understand what spatial mapping has to do with interactions.